src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel.mdx

```diff
@@ -18,19 +18,14 @@ import k8sOTelDiagram from 'images/infrastructure_diagram_k8s-otel-stack.webp'
 </Callout>
 
 
-Kubernetes is an open-source tool for automating deployments, scaling, and managing containerized applications. The New Relic Kubernetes monitoring integration gives you a quick and easy way to see what's going on with your Kubernetes clusters and workloads, whether they're hosted on-premises or in the cloud.
 
-This document outlines the process for monitoring a Kubernetes cluster using OpenTelemetry. It involves the installation of the [`nr-k8s-otel-collector`](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector) Helm chart within the cluster and the deployment of the necessary collectors to enable first-class observability.
+OpenTelemetry observability for Kubernetes offers users comprehensive, open-source instrumentation paired with first-class Kubernetes UI that our proprietary Kubernetes instrumentation already supports. Our K8s UIs  UIs are designed to be provider agnostic, giving users complete flexibility to choose between OpenTelemetry or New Relic instrumentation based on their needs.
```

Comment:
Our K8s **UIs  UIs** are designed to be provider agnostic : UIs is written twice.

Suggestion:  .... to choose between OpenTelemetry or New Relic instrumentation based on **your requirements**.

Reply:
I've rewritten it. Thank you.